html-lists/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Lists</title>
+  </head>
+  <body>
+    <h1>HTML Lists</h1>
+    <p><h2>Ordered List Example</h2></p>
+    <p>
+      <ol>
+         <li>Learn to code.</li>
+         <li>Build Projects.</li>
+         <li>Get a job.</li>
+    </ol>
+  </p>
+  <p><h2>Unordered List Example</h2></p>
```

Comment:
These elements don't need to be children of p tags, if you want to put them in a container, you can put everything in a `<div>` but they do not request individual containers withour reason.